apps/ai/src/utils/agent.ts

```diff
@@ -13,7 +13,7 @@ import type { User, UserList } from "../types/user";
 import type { WorkingHours } from "../types/workingHours";
 import now from "./now";
 
-const gptModel = "gpt-4";
+const gptModel = "gpt-4-0613";
```

Comment:
Is there any specific reason why we picked this Model exaclty? Additional context would be really helpful